src/main/resources/assets/patchouli/lang/de_DE.lang

```diff
@@ -0,0 +1,93 @@
+# Item Names
+item.patchouli:guide_book.name=Reiseführer
+
+# Testing
+item.patchouli:test_book_1.name=Testbuch 1
+item.patchouli:test_book_1.landing=Nicht so toll wie eine Einführung, es ist aber trotzdem präsent.
+
+item.patchouli:test_book_2.name=Testbuch 2
+item.patchouli:test_book_2.landing=Einführungstext! Es unterstützt auch $(2)Farben$() und $(4)solche $(bold)Sachen$()!
+
+item.patchouli:test_completion.name=Fortschritts-Testbuch
+item.patchouli:test_completion.landing=Das Buch ändert die Farben, wenn du Einträge freischaltest!
+
+item.patchouli:intro_book.name=Patchouli
+item.patchouli:intro_book.subtitle=Dokumentation für jedermann
+item.patchouli:intro_book.landing=Mit $(item)Patchouli$() machst du es dir einfach, fortschrittliche freischaltbare $(thing)Bücher$(), zu lesen für Modifikationen und Modifikationenpakete!
+
+
+#Subtitles
+patchouli.subtitle.book_open=Buch, das geöffnet wird
+patchouli.subtitle.book_flip=Seitenumschläge
+
+
+
+# Gui Strings
+patchouli.gui.lexicon.landing_info=(Platzhalter für den Einführungstext, bestimme dein eigenes in der json-Datei deines Buches!)
+patchouli.gui.lexicon.progress_meter=Fortschritt freischalten:
+patchouli.gui.lexicon.progress_tooltip=%d/%d Freigeschaltet
+patchouli.gui.lexicon.progress_tooltip.secret1=Plus 1 Geheimnis
+patchouli.gui.lexicon.progress_tooltip.secret=Plus %d Geheimnisse
+patchouli.gui.lexicon.progress_tooltip.info=Mache Fortschritte um noch mehr freizuschalten!
```

Comment:
This is how Vanilla translates it. Please stay close to the Vanilla translation where appropriate.
```suggestion
patchouli.gui.lexicon.progress_tooltip.info=Erziele Fortschritte, um noch mehr freizuschalten!
```

Reply:
I know. I missed it.